devito/ops/types.py

```diff
@@ -30,7 +71,7 @@ class OpsAccessible(basic.Symbol):
         to ``np.float32``.
     """
 
-    is_Scalar = True
+    is_Scalar = False
```

Comment:
why?

Reply:
this flag serves only for differentiating between `ops_arg_gbl` and `ops_arg_dat` in the method `create_ops_arg`
https://github.com/opesci/devito/blob/master/devito/ops/transformer.py#L274

moreover, an `OpsAccessible` represents a small array